### PR TITLE
Add missed projects

### PR DESCRIPTION
**Pull request by kgusakov**
Added missed GCE projects

### CODE REVIEW

File: google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineServiceAdapter.java [hunk 1]
```diff
@@ -286,15 +280,30 @@ public MachineTypeInZone apply(MachineType arg0) {
...
+      if ((image = api.getImageApiForProject(userProject.get()).get(id)) != null) return image;
+      if ((image = api.getImageApiForProject(DEBIAN_PROJECT).get(id)) != null) return image;
+      if ((image = api.getImageApiForProject(CENTOS_PROJECT).get(id)) != null) return image;
+      if ((image = api.getImageApiForProject(UBUNTU_PROJECT).get(id)) != null) return image;
+      if ((image = api.getImageApiForProject(GOOGLE_CONTAINERS_PROJECT).get(id)) != null) return image;
+      if ((image = api.getImageApiForProject(RHEL_PROJECT).get(id)) != null) return image;
+      if ((image = api.getImageApiForProject(CORE_OS_PROJECT).get(id)) != null) return image;
+      if ((image = api.getImageApiForProject(OPENSUSE_PROJECT).get(id)) != null) return image;
+      if ((image = api.getImageApiForProject(SUSE_PROJECT).get(id)) != null) return image;
+      if ((image = api.getImageApiForProject(WINDOWS_PROJECT).get(id)) != null) return image;
```
Comment (netvl) [inline google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineServiceAdapter.java]: I think it's better to use [ObjectUtils.firstNonNull](http://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/ObjectUtils.html#firstNonNull%28T...%29) if jclouds-labs-google depends on commons-lang, which is probable. If it does not, then this is fine.

**Comment by netvl** [comment]
Looks good to me.

File: google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineServiceAdapter.java (hunk 1 above)
Comment (netvl) [inline google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineServiceAdapter.java]: Or even better, http://stackoverflow.com/a/26118140/788207

Comment (kgusakov) [inline google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineServiceAdapter.java]: Yep, I thought about this horrible piece of code. But in jclouds-labs-google: 
- no commons-lang in dependencies
- Java 7 version

Comment (netvl) [inline google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineServiceAdapter.java]: Well, _we_ use Java 8 only, so it doesn't really matter.

Comment (kgusakov) [inline google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineServiceAdapter.java]: It doesn't matter what we use, I mean that jclouds parent use:

```
<maven.compile.source>1.7</maven.compile.source>
<maven.compile.target>1.7</maven.compile.target>
```

I'm not sure that we really want to update parent pom of jclouds for this small improvement.

Comment (netvl) [inline google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineServiceAdapter.java]: Ah, ok. In that caseit's ok.

File: google-compute-engine/src/test/java/org/jclouds/googlecomputeengine/features/ImageApiExpectTest.java [hunk 2]
```diff
@@ -68,6 +68,83 @@
...
+                   .payload(staticPayloadFromResource("/image_list_empty.json")).build();
+
+   public static final HttpRequest LIST_WINDOWS_IMAGES_REQUEST = HttpRequest
+           .builder()
+           .method("GET")
+           .endpoint("https://www.googleapis.com/compute/v1/projects/windows-cloud/global/images")
+           .addHeader("Accept", "application/json")
+           .addHeader("Authorization", "Bearer " + TOKEN).build();
+
+   public static final HttpResponse LIST_WINDOWS_IMAGES_RESPONSE =
```
Comment (netvl) [inline google-compute-engine/src/test/java/org/jclouds/googlecomputeengine/features/ImageApiExpectTest.java:144]: Are all these constants used anywhere?

Comment (netvl) [inline google-compute-engine/src/test/java/org/jclouds/googlecomputeengine/features/ImageApiExpectTest.java:144]: Ah, I see. It is the above file. Okay.

**Comment by netvl** [comment]
Looks good :+1:

**Pull request merged**